tests/integration/conftest.py

```diff
@@ -27,6 +27,8 @@ def maybe_get_id(obj):
         Therefore, always try to get the id of a parameter from
         pytest.mark.parametrize as a first option.
         """
+        if type(obj) == tuple:
+            return maybe_get_id(obj[0])
```

Comment:
Changed this because parametrizing with tuples in tests/integration/test_multiprocess.py